gobstones-spec.gemspec

```diff
@@ -0,0 +1,25 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'gobstones/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'gobstones-spec'
+  spec.version       = Gobstones::VERSION
+  spec.authors       = ['Federico Aloi']
+  spec.email         = ['federico.aloi@gmail.com']
```

Comment:
Haha, you have not being consistent here regarding attributions :stuck_out_tongue_closed_eyes: